src/abhijitMJun20/Bankingsystem.java

```diff
@@ -0,0 +1,52 @@
+package abhijitMJun20;
+
+public class Bankingsystem {
+	int accountbalance=1000;
```

Comment:
No Need to initialize this variable. perform credit() method first.

Reply:
Done